docs/how-to/install-pebble.md

```diff
@@ -16,15 +18,29 @@ To install the binary for the latest version of Pebble:
    :end-before: End: Install Pebble binary
 ```
 
+
+(install_pebble_snap)=
+## Install the snap
+
+To install the latest version of Pebble from the Snap Store:
+
+```
+sudo snap install pebble --classic
```

Comment:
Do we recommend installing snaps with `sudo`? Without `sudo`, you'll get Ubuntu prompting for the privileges (assuming you're using Ubuntu), which feels nicer to me (both in terms of use and in terms of what we recommend people do). On the other hand, if you're using Pebble, you're probably very familiar with the command line so maybe you want to stay confined there.

You can also install the snap by searching for pebble in the Ubuntu software app :)

Reply:
At first I was going to say I agree that it would be better to drop the `sudo`. However, the official snap [tutorial](https://snapcraft.io/docs/get-started#install-the-snap) uses it -- I guess the thinking is they're already in the terminal, so why not keep them there? So maybe we should leave it with `sudo` for consistency.